### PR TITLE
Simplifying sdk.sh

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: tools/sdk.sh [hunk 1]
```diff
@@ -142,23 +137,7 @@ build_project_if_requested ()
...
-        then
-            API_VERSION=`cat AndroidManifest.xml | grep minSdkVersion | cut -d"\"" -f2`
-        else
-            API_VERSION=$3
-        fi
-        android update project -p . -t "android-$API_VERSION" | grep "$BUILD_OUTPUT_FILTER"
-
-        if [ -z $4 ]
-        then
-            ant clean debug | grep "$BUILD_OUTPUT_FILTER"
```
Comment (bhariharan) [inline tools/sdk.sh]: Looks like I missed some ```ant``` references in my cleanup effort :-(

File: tools/sdk.sh [hunk 2]
```diff
@@ -182,20 +161,20 @@ else
...
-        build_project_if_requested    "AppConfigurator"          $NATIVE_TOP/SampleApps/AppConfigurator          23 :native:NativeSampleApps:AppConfigurator
-        build_project_if_requested    "ConfiguredApp"            $NATIVE_TOP/SampleApps/ConfiguredApp            23 :native:NativeSampleApps:ConfiguredApp
-        build_project_if_requested    "SmartSyncExplorer"        $NATIVE_TOP/SampleApps/SmartSyncExplorer        23 :native:NativeSampleApps:SmartSyncExplorer
-        build_project_if_requested    "AccountEditor"            $HYBRID_TOP/SampleApps/AccountEditor            23 :hybrid:HybridSampleApps:AccountEditor
-        build_project_if_requested    "NoteSync"                 $HYBRID_TOP/SampleApps/NoteSync                 23 :hybrid:HybridSampleApps:NoteSync
-        build_project_if_requested    "SmartSyncExplorerHybrid"  $HYBRID_TOP/SampleApps/SmartSyncExplorerHybrid  23 :hybrid:HybridSampleApps:SmartSyncExplorerHybrid
-        build_project_if_requested    "ForcePluginsTest"         $HYBRID_TOP/test/ForcePluginsTest               23 :hybrid:test:ForcePluginsTest
+        build_project_if_requested    "Cordova"                       :external:cordova:framework
```
Comment (bhariharan) [inline tools/sdk.sh]: Nice! So the ```targetApiVersion``` comes from ```AndroidManifest.xml``` or ```build.gradle``` I'm guessing.

**Comment by bhariharan** [comment]
:+1:

**Comment by kchitalia** [comment]
:+1:

**Pull request merged**